src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -3217,6 +3217,158 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorModel'
+  /api/repository/v1/snapshotAccessRequests/{id}/members:
+    get:
+      tags:
+        - SnapshotAccessRequest
+      description: Get the members in the DAC group
+      operationId: getSnapshotAccessRequestGroupMembers
+      parameters:
+        - $ref: '#/components/parameters/Id'
```

Comment:
I think the reuse of an existing object is a good idea, but Id specifies that the format of the string is a UUID and this will not match that format because it will be an email. So if that type doesn't exist yet maybe adding a new email type would be good

Reply:
I think this is the snapshot request ID, so isn't that a UUID?

Reply:
Yes, I think for the GET, it's passing in the UUID of the snapshot access request whose group members you want to get